docs/src/content/docs/examples/hn.mdx

```diff
@@ -1,14 +1,87 @@
 ---
-title: Hacker News sentiment analysis
-description: Analyzing Hacker News sentiment with GenSX
+title: Hacker News Analyzer
+description: Extract out key trends from top Hacker News posts
 sidebar:
   order: 1
 ---
 
-GenSX is a simple framework for building complex generative AI and LLM workflows.
+The [Hacker News Analyzer](https://github.com/cortexclick/gensx/tree/main/examples/hackerNewsAnalyzer) example uses GenSX to fetch, analyze, and extract trends from the top Hacker News posts. It shows how to combine data fetching, parallel analysis, and content generation in a single workflow to create two outputs: a detailed report and a tweet of the trends.
 
-It uses `jsx` to define and orchestrate workflows with functional, reusable components:
+## Workflow
+
+The Hacker News Analyzer workflow is composed of the following steps:
+
+1. Fetch the top 500 Hacker News posts (`<HnCollector>`)
+2. Process each post in parallel (`<AnalyzeHNPosts>`)
+   a. Summarize the content (`<PostSummarizer>`)
+   b. Analyze the comments (`<CommentsAnalyzer>`)
+3. Writes a detailed report identifying the key trends across all posts (`<TrendsAnalyzer>`)
+4. Edits the report into the style of Paul Graham (`<PGEditor>`)
+5. Generates a tweet in the voice of Paul Graham (`<PGTweetWriter>`)
```

Comment:
Both of these steps in the workflow don't do a particularly good job at emulating paul graham. Might be worth following up with a tweak to those two examples that uses few shot prompting whenever we have time.

Reply:
sounds good, i agree it makes sense to rev on these before we launch.